client/packages/invoices/src/StockOut/useAllocationContext.ts

```diff
@@ -237,7 +239,7 @@ export const useAllocationContext = create<AllocationContext>((set, get) => ({
       item?.doses ?? 0
     );
 
-    autoAllocate(quantityInNewPackSize, format, t);
+    autoAllocate(quantityInNewPackSize, format, t, 0);
```

Comment:
I'm not super happy about just using `0` here, but it's a real hassle to actually make the `expiryThresholdDays` available here (I tried making it available to Context, but it was very fiddly and not working reliably), and I couldn't find any cases where it actually mattered here (it seems to always use the `autoAllocate` method directly rather than this `reallocateLines` method when actually allocating in the UI, so I think we're safe.